### ISSUE TITLE
Issue with the extract_staypoints function

### ISSUE THREAD

**Issue by ShozenD** [mie-lab/trackintel#74]
Hi! I'am an undergraduate student at Keio University in Japan, and I'm hoping to use this package for analyzing trajectory data in Tokyo. I'm also hoping to contribute to its development and make it more generalizable!

I was reading the tutorials section of the documentation and I found an issue with the `extract_staypoints` function.

The data you provide as the example does not have elevation data. As a result, 
```{python}
staypoint['elevation'] = np.mean([pfs[k]['elevation'] for k in range(i, j)]) # around line 106
```
and
```{python}
staypoint['elevation'] = pfs[j]['elevation'] # around line 126
```
returns a key error.

Additionally, in the documentation under tutorial, the `time_threshold` is too large for the dataset and no staypoints are detected (an appropriate threshold might be 60). Also, in the second line there is a typo
```{python}
stps = pfs.as_positionfixes.extract_staypoints(method='sliding', dist_threshold=100, time_threshold=5*60)
spts.as_staypoints.plot(out_filename='staypoints.png', radius=meters_to_decimal_degrees(100, 47.5), positionfixes=pfs, plot_osm=True)
```
I believe the `spts.as_staypoints.plot` should be changed to `stps.as_staypoints.plot`

**Comment by henrymartin1** [comment]
Hi Shozen,
I am very happy to hear that you are interested in using trackintel for your project. We are currently working on finishing the core functionalities until end of this year. Because of this, there were a lot of changes in the model and the library in the last weeks, so I am honestly not surprised that you found something that wasn't quite right.
That being said, thanks for taking the time for fixing it, I will look at the pull request right away!

If you are interested, we would be very happy if you want to contribute more to the code base or also if you would like to create an example based on your current work about the trajectory data in Tokyo.

Best,
Henry

**Comment by henrymartin1** [comment]
closed with #75
